.github/actions/sync/shared-config.rb

```diff
@@ -98,6 +99,10 @@ def git(*args)
   rubydoc.brew.sh
   ruby-macho
 ].freeze
+custom_actionlint_repos = %w[
+  brew
+  homebrew-core
```

Comment:
What's custom in the actionlint workflow for these two?

Reply:
They reuse the clones made by `setup-homebrew` (and `homebrew-core` has `core: true` there). The `brew` workflow also runs on changes to `.github/actionlint.yaml` (which is different from `.github/workflows/actionlint.yml`). `homebrew-core` has some handling for `shell: /bin/bash`.